README.md

```diff
@@ -34,6 +52,23 @@ use greptime_proto::v1::meta::*;
 use greptime_proto::prometheus::remote::*;
 ```
 
+### Go
+
+Download the go module:
+
+```console
+go get github.com/GreptimeTeam/greptime-proto@main
+```
+
+Then use greptime-proto as the normal Go module:
+
+```go
+import (
+        greptimev1 "github.com/GreptimeTeam/greptime-proto/go/greptime/v1"
```

Comment:
Is the package path verbose? like there are three `greptime` in the package path

Reply:
I think it's the style issue. For now, the package path is consistent with the directory structure of `./proto`.

For example:

The `meta/*.proto` is in `./proto/greptime/v1/meta/`, then the generated `meta` go package will use `${ModuleRoot}/go/greptime/v1` as its path.